src/Resources/Resources/help/New-AzPolicyAssignment.md

```diff
@@ -59,7 +59,18 @@ Specify a policy and scope.
 
 ## EXAMPLES
 
-### Example 1: Policy assignment at resource group level
+### Example 1: Policy assignment at subscription level
+```
+PS C:\> $Subscription = Get-AzSubscription -SubscriptionName 'Subscription01'
+PS C:\> $Policy = Get-AzPolicyDefinition -Name 'VirtualMachinePolicy'
+PS C:\> New-AzPolicyAssignment -Name 'VirtualMachinePolicyAssignment' -PolicyDefinition $Policy -Scope "/subscriptions/$($Subscription.Id)"
```

Comment:
The typical way of acting on a subscription in powershell is by using "Select-AzSubscription" then just calling New-AzPolicyAssignment without any further scoping info (i.e. just with -Name and -PolicyDefinition) in the above example.  It uses subscription scope by default (it should at least).

Select-AzSubscription sets your global subscription context for all cmdlets.

Reply:
'Scope' is required. 
New-AzPolicyAssignment : Cannot validate argument on parameter 'Scope'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.